[FR] Add `Medals/Unlock Requirements/Beatmap Challenge Packs`, `Beatmap Packs`, `Beatmap Spotlights`, `Dedication`, `Seasonal Spotlights`, `Skill` (#5889)
ppy/osu-wiki: wiki/Medals/Unlock_requirements/Skill/fr.md

# Conditions de déverrouillage des médailles : Skill

| Nom de la médaille | Condition |
| :-- | :-- |

> | 500 Combo | Atteindre 500 de combo sur une beatmap de n'importe quel mode de jeu. |

Reviewer: ```suggestion
| 500 Combo | Atteindre un combo de 500 sur une beatmap de n'importe quel mode de jeu. |
```

> | 750 Combo | Atteindre 750 de combo sur une beatmap de n'importe quel mode de jeu. |

Reviewer: ```suggestion
| 750 Combo | Atteindre un combo de 750 sur une beatmap de n'importe quel mode de jeu. |
```

> | 1000 Combo | Atteindre 1000 de combo sur une beatmap de n'importe quel mode de jeu. |

Reviewer: ```suggestion
| 1000 Combo | Atteindre un combo de 1000 sur une beatmap de n'importe quel mode de jeu. |
```

> | 2000 Combo | Atteindre 2000 de combo sur une beatmap de n'importe quel mode de jeu. |

Reviewer: ```suggestion
| 2000 Combo | Atteindre un combo 2000 sur une beatmap de n'importe quel mode de jeu. |
```

| Building Confidence (osu!)/Not Even Trying (osu!taiko)/Zesty Disposition (osu!catch)/Impulse Drive (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 3,00★ et 3,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |
| Insanity Approaches (osu!)/Face Your Demons (osu!taiko)/Hyperdash ON! (osu!catch)/Hyperspeed (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 4,00★ et 4,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

> | These Clarion Skies (osu!)/The Demon Within (osu!taiko)/It's Raining Fruit (osu!catch)/Ever Onwards (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 5,00★ et 5,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| These Clarion Skies (osu!)/The Demon Within (osu!taiko)/It's Raining Fruit (osu!catch)/Ever Onwards (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 5,00★ et 5,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

| Above And Beyond (osu!)/Drumbreaker (osu!taiko)/Fruit Ninja (osu!catch)/Another Surpassed (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 6,00★ et 6,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

> | Supremacy (osu!)/The Godfather (osu!taiko)/Dreamcatcher (osu!catch)/Extra Credit (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 7,00★ et 7,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Supremacy (osu!)/The Godfather (osu!taiko)/Dreamcatcher (osu!catch)/Extra Credit (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 7,00★ et 7,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

| Absolution (osu!)/Rhythm Incarnate (osu!taiko)/Lord of the Catch (osu!catch)/Maniac (osu!mania) | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 8,00★ et 8,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |
| Event Horizon | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 9,00★ et 9,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

> | Phantasm | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 10,00★ et 10,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Phantasm | Terminer une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 10,00★ et 10,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Totality (osu!)/Keeping Time (osu!taiko)/Sweet And Sour (osu!catch)/Keystruck (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 1,00★ et 1,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Totality (osu!)/Keeping Time (osu!taiko)/Sweet And Sour (osu!catch)/Keystruck (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 1,00★ et 1,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Business As Usual (osu!)/To Your Own Beat (osu!taiko)/Reaching The Core (osu!catch)/Keying In (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 2,00★ et 2,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Business As Usual (osu!)/To Your Own Beat (osu!taiko)/Reaching The Core (osu!catch)/Keying In (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 2,00★ et 2,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Building Steam (osu!)/Big Drums (osu!taiko)/Clean Platter (osu!catch)/Hyperflow (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 3,00★ et 3,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Building Steam (osu!)/Big Drums (osu!taiko)/Clean Platter (osu!catch)/Hyperflow (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 3,00★ et 3,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Moving Forward (osu!)/Adversity Overcome (osu!taiko)/Between The Rain (osu!catch)/Breakthrough (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 4,00★ et 4,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Moving Forward (osu!)/Adversity Overcome (osu!taiko)/Between The Rain (osu!catch)/Breakthrough (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 4,00★ et 4,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Paradigm Shift (osu!)/Demonslayer (osu!taiko)/Addicted (osu!catch)/Everything Extra (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 5,00★ et 5,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Paradigm Shift (osu!)/Demonslayer (osu!taiko)/Addicted (osu!catch)/Everything Extra (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 5,00★ et 5,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Anguish Quelled (osu!)/Rhythm's Call (osu!taiko)/Quickening (osu!catch)/Level Breaker (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 6,00★ et 6,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Anguish Quelled (osu!)/Rhythm's Call (osu!taiko)/Quickening (osu!catch)/Level Breaker (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 6,00★ et 6,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Never Give Up (osu!)/Time Everlasting (osu!taiko)/Supersonic (osu!catch)/Step Up (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 7,00★ et 7,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée.|

Author: ```suggestion
| Never Give Up (osu!)/Time Everlasting (osu!taiko)/Supersonic (osu!catch)/Step Up (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 7,00★ et 7,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |
```

> | Never Give Up (osu!)/Time Everlasting (osu!taiko)/Supersonic (osu!catch)/Step Up (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 7,00★ et 7,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Never Give Up (osu!)/Time Everlasting (osu!taiko)/Supersonic (osu!catch)/Step Up (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 7,00★ et 7,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Aberration (osu!)/The Drummer's Throne (osu!taiko)/Dashing Scarlet (osu!catch)/Behind The Veil (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 8,00★ et 8,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Aberration (osu!)/The Drummer's Throne (osu!taiko)/Dashing Scarlet (osu!catch)/Behind The Veil (osu!mania) | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 8,00★ et 8,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu correspondant. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Chosen | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 9,00★ et 9,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Chosen | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 9,00★ et 9,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```

> | Unfathomable | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 10,00★ et 10,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulty-increase), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulty-reduction) n'est pas autorisée. |

Reviewer: ```suggestion
| Unfathomable | Réaliser un full combo sur une beatmap qualifiée, classée ou approuvée qui se situe dans un star rating entre 10,00★ et 10,99★ (après que tous les [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée), s'il y en a, aient été pris en compte) dans le mode de jeu osu!. Toute utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) n'est pas autorisée. |
```
